SteamBundleSitesExtension.js

```diff
@@ -891,30 +955,11 @@ const siteHandlers = {
             bundleSitesBoxHandler.reveal(handler, $(source).find('a[id^=fetchlink_]'));
         });
         $('.SBSE_BtnRetrieve').click(() => {
-            const keys = [];
-
-            $(source).find('.game-key-string').each((index, element) => {
-                const $ele = $(element);
-                const key = $ele.find('.keys').val();
-
-                if (key) {
-                    const $a = $ele.find('.title_game > a');
-                    const title = $a.text().trim();
-
-                    // append key details
-                    keyDetails.set(key, {
-                        url: $a.attr('href'),
-                        title: $a.text(),
-                    });
-
-                    keys.push({
-                        key,
-                        title,
-                    });
-                }
-            });
-
-            bundleSitesBoxHandler.retrieve(keys);
+            bundleSitesBoxHandler.retrieve(extractKeys());
+        });
+        $('.SBSE_BtnExport').click(() => {
+            let bundleTitle = 'IndieGala ' + $('#steam-key').prev()[0].innerText;
```

Comment:
`const` when not changing variable and [https://eslint.org/docs/rules/prefer-template](url)

Reply:
Got it~ XD